packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -68,7 +68,7 @@ TODO: Remove this section if there are not any general updates.
 
 ## Deep links tool updates
 
-TODO: Remove this section if there are not any general updates.
+- Added support for validating ios deep link settings.
```

Comment:
can you add a screenshot and a link to this PR?

Reply:
added, thanks!